src/applications/mhv-medical-records/tests/components/LabsAndTestsListItem.unit.spec.jsx

```diff
@@ -196,11 +196,16 @@ describe('LabsAndTestsListItem component with pathology record', () => {
   });
 
   it('should display the name of the record as a link to view record details', () => {
-    const recordName = screen.getAllByText('LR SURGICAL PATHOLOGY REPORT', {
-      selector: 'a',
+    // matches any one of the four expected strings
+    const pathologyLabelRe = /^LR SURGICAL (PATHOLOGY|SURGICAL_PATHOLOGY|ELECTRON_MICROSCOPY|CYTOPATHOLOGY) REPORT$/;
+
+    const recordNames = screen.getAllByText(pathologyLabelRe, {
       exact: true,
-    })[0];
-    expect(recordName).to.exist;
+      selector: 'a',
+    });
+
+    // ensure at least one matching link was found
+    expect(recordNames.length).to.be.greaterThan(0);
```

Comment:
This test is not really checking anything new. The fixture hasn't changed so I don't think this is necessary. Same for the other two tests you modified.

Instead of modifying these tests, I think you should add a new `describe('convertPathologyRecord', () => {` block to reducers/labsAndTests.unit.spec.jsx. You should just be able to build a quick test for the name without using any fixtures.

Reply:
fixed